server/api/impl/admin/synchronize.go

```diff
@@ -180,7 +113,127 @@ func (adm *Admin) UninstallApp(appID apps.AppID) error {
 	return nil
 }
 
-func (adm *Admin) RegisterApp(manifest *apps.Manifest) error {
+func getAppsForRegistration(path string) (apps.AppVersionMap, error) {
```

Comment:
What does the `ForRegistration` part of the name mean?

Reply:
App that are "listed" in the marketplace are called "registered" there status is also "registered". We also have "installed" and "uninstalled" apps